Automattic/wp-calypso
# Domain Redirect rename to Domain Forwarding (copies)

Related to https://github.com/Automattic/dotcom-forge/issues/3515

## Proposed Changes

Change all text related to Domain Redirect and rename it to Domain Forwarding.

## Testing Instructions

* Edit a domain
* Open the Forwarding domain card
* Check if all the copies are good and makes sense.

## TO DO
- Add fallback to show translations mentioning Domain Redirect

## Pre-merge Checklist
